src/main/java/ch/uzh/ifi/hase/soprafs22/service/LobbyService.java

```diff
@@ -41,10 +40,16 @@ public LobbyService(UserRepository userRepository) {
         this.lobbyManager = LobbyManager.getInstance();
     }
 
-    private void checkLobbyNameNullOrEmpty(String lobbyName, String errorMessageEnding) {
-        if (lobbyName == null || lobbyName.isEmpty()) {
-            String errorMessage = "The lobby name provided is empty. Therefore, the lobby could not be " + errorMessageEnding + "!";
-            throw new ResponseStatusException(HttpStatus.BAD_REQUEST, errorMessage);
+    private void checkStringConfigNullOrEmpty(String s, String errorMessageEnding, boolean isToken) {
```

Comment:
I would do two methods one for token and one for the other things.
Also now the error message is always "The lobby name provided....", I would add a parameter for the whole error message.

Reply:
Or do it even more general and allow to set the httpstatus too